Bruin Bite/UserClass.swift

```diff
@@ -73,7 +73,9 @@ public class User {
         else if ((user_error_model?.error_description ?? nil) != nil) {
             net_error = (user_error_model?.error_description)!
         }
-        print(net_error)
+        if(net_error != "") {
```

Comment:
Can remove the check for empty, since our logger function already checks for that.

Reply:
If net error is empty in this case, Logger will still log "Warning: Net Error:" because message is not empty. Which means we will be getting warning messages every single time we use this update error function. I put the empty check in the Logger class because an empty error should never be logged, not to handle cases like this.

Reply:
Okay cool, that sounds good!